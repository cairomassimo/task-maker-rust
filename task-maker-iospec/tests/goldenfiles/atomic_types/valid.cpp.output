32705 2 160 
