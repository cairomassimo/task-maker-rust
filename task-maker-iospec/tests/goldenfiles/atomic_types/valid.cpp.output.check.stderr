error: invalid atomic value
  --> IOSPEC:6:10
   |
 1 | inputln {
 2 |     item xi32: i32;
 3 |     item xi64: i64;
 4 |     item xbool: bool;
...
 9 |     item yi64: i64;
10 |     item ybool: bool;
   |          ----- info: when reading this
   |
